policies.go

```
@@ -96,7 +96,8 @@ func (c *cowHostList) remove(ip net.IP) bool {
 	found := false
 	newL := make([]*HostInfo, 0, size)
 	for i := 0; i < len(l); i++ {
-		if !l[i].ConnectAddress().Equal(ip) {
+		connAddr, _ := l[i].ConnectAddress()
+		if !connAddr.Equal(ip) {
```

`add()` should check if `.ConnectAddress()` returns an error and if it does it shouldn't add it. If we do that then yeah we can ignore the error here because it's guaranteed to always be `nil`

---

I've add the check to the `add()`